### PR TITLE
Use MPU6050 and Drivetrain from Lib2585, update unit tests, and add n…

### PR DESCRIPTION
**Pull request by jwong101**
…ew Lib2585

### CODE REVIEW

**Comment by jwong101** [comment]
Closes #100, #102, and #101

**Comment by LoadingPleaseWait** [comment]
Many of the instance variables in `WheelSystem.java` should now be removed.

File: src/org/impact2585/frc2016/systems/WheelSystem.java [hunk 1]
```diff
@@ -181,86 +189,51 @@ public void initializeForwardPID(double setpoint) {
...
-		return currentRampForward;
+		return drivetrain.getCurrentRampForward();
 	}
 	
 	/**sets the current ramp forward to rampForward
 	 * @param rampForward the distance the robot is moving forward
 	 */
 	public void setCurrentRampForward(double rampForward) {
-		currentRampForward = rampForward;
+		drivetrain.setCurrentRampForward(rampForward);;
```
Comment (LoadingPleaseWait) [inline src/org/impact2585/frc2016/systems/WheelSystem.java]: It doesn't get much more minor than this but you have two semicolons.

File: src/org/impact2585/frc2016/systems/WheelSystem.java [hunk 2]
```diff
@@ -181,86 +145,51 @@ public void initializeForwardPID(double setpoint) {
 	 * @param rotate how much the robot will turn
 	 */
 	public void drive(double driveForward, double rotate) {
-		drivetrain.arcadeDrive(driveForward, -rotate);
+		drivebase.arcadeDrive(driveForward, -rotate);
```
Comment (LoadingPleaseWait) [inline src/org/impact2585/frc2016/systems/WheelSystem.java:148]: If you passed true into the constructor of `Drivetrain`, would this mean that you're inverting rotation twice?

**Comment by jwong101** [comment]
No, since that drive method is only being used for the PID subsystem.

**Comment by LoadingPleaseWait** [comment]
Alright then

**Pull request merged**